src/components/StatusLabel/StatusLabel.tsx

```diff
@@ -56,8 +57,8 @@ const Wrapper = styled.span<{ themes: Theme }>`
       }
 
       &.process {
-        border-color: ${color.WARNING};
-        color: ${color.WARNING};
+        border-color: #ffcc17;
```

Comment:
Hmm I got that these colors are used only here so no need to constantize (and agree with that 👍 ).